src/style.css

```diff
@@ -1,3 +1,53 @@
 body {
   margin: 0;
+  font-family: 'Roboto', sans-serif;
+  background-color: lightgray;
+}
+
+header {
+  align-items: center;
+  display: flex;
+  justify-content: space-between;
+  padding: 5px 50px;
+  background-color: rgba(255, 255, 255, 1);
+}
+
+ul {
+  display: flex;
+  list-style: none;
+}
+
+li {
+  margin: 0 10px;
+  font-size: small;
+  font-weight: bold;
+}
```

Comment:
Be consistent with your vertical margins. You're adding both top and bottom margins to the 'li' elements. It's recommended to add only top or only bottom margin to maintain consistency and avoid margin collapse issues.